Packs/qualys/Integrations/Qualysv2/Qualysv2.yml

```diff
@@ -546,6 +546,17 @@ script:
       name: cloud_tag_fields
     - description: Specify a positive numeric value to limit the amount of results in the requested list
       name: limit
+      auto: PREDEFINED
+    - auto: PREDEFINED
+      defaultValue: Basic
+      description: '(Optional) Show the requested amount of host information for each host. A valid value is: Basic, Basic/AGs, All, All/AGs, or None.'
+      name: details
+      predefined:
+      - Basic
+      - Basic/AGs
```

Comment:
Can you specify somewhere (maybe in description) that AG = Assets Groups, please

Reply:
Updated